src/webui/api/rsscontroller.h

```diff
@@ -44,9 +44,11 @@ private slots:
     void removeItemAction();
     void moveItemAction();
     void itemsAction();
+    void markAsReadAction();
```

Comment:
Looks like a const function to me.

Reply:
Let's keep it consistent with other methods.
Or you can fix them all in separate PR.

Reply:
>Let's keep it consistent with other methods.

What is the value in "keep it consistent"? 

>Or you can fix them all in separate PR.

As if I have nothing better to do... IMO it should be done in this PR properly, but if you insist then never mind...

Reply:
>What is the value in "keep it consistent"?

If you add `const` only to this single "action", it will be confusing, giving the false impression that this one is different from other "actions".
>IMO it should be done in this PR properly, but if you insist then whatever...

It can be done even here, but for all similar methods.

Reply:
I hope i changed the correct functions to const members. I also changed the other two occurrences of "\n" with '\n' in appcontroller.cpp.